### PR TITLE
About me and other pages

### PR DESCRIPTION
**Pull request by monika-soam**
- Added "Works" section, "About myself" section, and all the mobile skeleton pages
- "Works" section used CSS Grid and Flexbox combined
- Use ul and li for categories in the "About myself" section. Languages, Frameworks, and Skills
- Used correct GitHub Flow
- Fixed linter error
- Added one outer div section and put the rest div inside the section to create a grid as per the suggestion.

### CODE REVIEW

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @monika-soam,

 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -31,5 +33,156 @@ <h3>LET’S CONNECT</h3>
...
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">javaScript</a></li>
+                </ul>
+            </div>
+            <button type="button">See Project</button>
+        </div>
+        
+    </div>
+
+</section><br><br>
```
[the remark this answers is not in the document]
Comment (monika-soam) [reply index.html]: Created an outer section and put all the div inside it to perform grid.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 # My Portfolio
```
Comment (Amrhub) [inline README.md:3]: - [OPTIONAL] I would suggest adding steps on how to get a copy of the project up and running on the local machine just as per the Microverse READme.MD file template

**Review by Amrhub — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @monika-soam, 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights ✅ 
✔️  Great communication with the previous code reviewer
✔️  Correct workflow and linters are passing
✔️  Clean code 👏🏻 

### Optional suggestion ♻️ 
- [OPTIONAL] I would recommend adding steps on how to set up the project locally just as per the Microverse template.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**